src/util.c

```diff
@@ -451,7 +451,7 @@ jv jq_util_input_next_input(jq_util_input_state *state) {
   return value;
 }
 
-#if defined(WIN32) && !defined(HAVE_STRPTIME)
+#ifndef HAVE_STRPTIME
```

Comment:
Was added in https://github.com/jqlang/jq/commit/fdab39bc7b8d41c1ae410f03a42afc10a9322c99

@nicowilliams do you remember if there was some reason for provide replacement strptime only for WIN32?

Reply:
There's no reason.  We could provide one for all platforms.  I just didn't want to have to do the work to make sure we'd not break anyone.